ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -103,7 +103,8 @@ Wait For RHODS Dashboard To Load
     IF    "${expected_page}" == "${NONE}"
         Wait Until Page Contains Element    //div[@data-testid="home-page"]    timeout=${timeout}
     ELSE
-        Wait For Dashboard Page Title    ${expected_page}    timeout=${timeout}
+        Wait Until Keyword Succeeds    3x    5s
+        ...    Wait For Dashboard Page Title    ${expected_page}    timeout=${timeout}
```

Comment:
I'm not sure this is going to help for the problem you're describing. If the issue is that the page takes longer than expect to get from "loading" state to "loaded" state, we either need to improve the checks in the `Wait...` keyword or increase the timeout. 
Adding retries here is likely to result in multiple failed checks for the same reason, no?

Reply:
Nevermind, the keyword does not reload the page in between retries, this should be good enough

Reply:
Yeah, there is no reload of the page. It's just restart of the check that expects some text value in the xpath element. This seemed like the easiest solution for this problem to me.